benchexec/runexecutor.py

```diff
@@ -829,19 +840,20 @@ def _execute(
                     # Disable energy measurements because we use only parts of a CPU
                     packages = None
 
-        def preParent():
+        def preParent(**kwargs):
```

Comment:
I think we should use proper parameters here. This is called only internally, so we do not need to pass through an unknown number of parameters.